kbs/docs/config.md

```diff
@@ -144,13 +144,14 @@ attestation. The following properties can be set.
 
 >gRPC CoCo AS is available only when `coco-as-grpc` feature is enabled.
```

Comment:
There's a minor glitch in the doc that could be addressed in a separate commit in this PR:
```suggestion
>Intel Trust Authority AS is available only when `intel-trust-authority-as` feature is enabled.
```

Reply:
Fixed.